docs/data/material/components/autocomplete/GitHubLabel.tsx

```diff
@@ -56,51 +62,63 @@ function PopperComponent(props: PopperComponentProps) {
 }
 
 const StyledPopper = styled(Popper)(({ theme }) => ({
-  border: `1px solid ${theme.palette.mode === 'light' ? '#e1e4e8' : '#30363d'}`,
-  boxShadow: `0 8px 24px ${
-    theme.palette.mode === 'light' ? 'rgba(149, 157, 165, 0.2)' : 'rgb(1, 4, 9)'
-  }`,
+  border: `1px solid ${'#e1e4e8'}`,
+  boxShadow: `0 8px 24px ${'rgba(149, 157, 165, 0.2)'}`,
+  color: '#24292e',
+  backgroundColor: '#fff',
   borderRadius: 6,
   width: 300,
   zIndex: theme.zIndex.modal,
   fontSize: 13,
-  color: theme.palette.mode === 'light' ? '#24292e' : '#c9d1d9',
-  backgroundColor: theme.palette.mode === 'light' ? '#fff' : '#1c2128',
+  ...theme.applyStyles('dark', {
+    border: `1px solid ${'#30363d'}`,
+    boxShadow: `0 8px 24px ${'rgb(1, 4, 9)'}`,
+    color: '#c9d1d9',
+    backgroundColor: '#1c2128',
+  }),
 }));
 
 const StyledInput = styled(InputBase)(({ theme }) => ({
   padding: 10,
   width: '100%',
-  borderBottom: `1px solid ${
-    theme.palette.mode === 'light' ? '#eaecef' : '#30363d'
-  }`,
+  borderBottom: `1px solid ${'#30363d'}`,
   '& input': {
     borderRadius: 4,
-    backgroundColor: theme.palette.mode === 'light' ? '#fff' : '#0d1117',
+    backgroundColor: '#0d1117',
+    border: `1px solid ${'#30363d'}`,
     padding: 8,
     transition: theme.transitions.create(['border-color', 'box-shadow']),
-    border: `1px solid ${theme.palette.mode === 'light' ? '#eaecef' : '#30363d'}`,
     fontSize: 14,
     '&:focus': {
-      boxShadow: `0px 0px 0px 3px ${
-        theme.palette.mode === 'light'
-          ? 'rgba(3, 102, 214, 0.3)'
-          : 'rgb(12, 45, 107)'
-      }`,
-      borderColor: theme.palette.mode === 'light' ? '#0366d6' : '#388bfd',
+      boxShadow: `0px 0px 0px 3px ${'rgba(3, 102, 214, 0.3)'}`,
+      borderColor: '#0366d6',
+      ...theme.applyStyles('dark', {
+        boxShadow: `0px 0px 0px 3px ${'rgb(12, 45, 107)'}`,
+        borderColor: '#388bfd',
+      }),
     },
+    ...theme.applyStyles('dark', {
+      backgroundColor: '#fff',
+      border: `1px solid ${'#eaecef'}`,
+    }),
   },
+  ...theme.applyStyles('dark', {
```

Comment:
I feel like we should group light and dark values to be right next to each other, while we should think about light mode and dark mode standalone, in which world I find https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark a harmful API. I believe it's still more important to group dark/light style per selector, to so people can have a simple mental model.